test/test_common/utility.h

```diff
@@ -211,44 +211,60 @@ class TestUtility {
   static Stats::GaugeSharedPtr findGauge(Stats::Store& store, const std::string& name);
 
   /**
-   * Wait till Counter value is equal to the passed ion value.
+   * Wait for a counter to == a given value.
    * @param store supplies the stats store.
    * @param name supplies the name of the counter to wait for.
    * @param value supplies the value of the counter.
    * @param time_system the time system to use for waiting.
+   * @param timeout the maximum time to wait before timing out.
```

Comment:
Document that if greater than zero, the maximum time...

Should this have a default? Or are all uses of this through integration/server_stats.h?

Reply:
Good catch, should have a default thanks